src/core/qgsunittypes.cpp

```diff
@@ -1624,6 +1624,7 @@ double QgsUnitTypes::fromUnitToUnitFactor( QgsUnitTypes::VolumeUnit fromUnit, Qg
       return 1.0;
     }
   }
+  return 1.0;
```

Comment:
Is that change intentional?

Reply:
yeah, it was just throwing up a warning during compiling